index.html

```diff
@@ -814,150 +814,156 @@ <h2>Privacy Considerations</h2>
 
     <section class="appendix">
       <h2>Test Vectors</h2>
-      <p>
-The following test vectors are provided to assist with implementers.
-      </p>
-
-      <pre class="example">
+      <section>
+        <h3>Representation: Ed25519Signature2020</h3>
+        <p>
+The signer needs to generate a private/public key pair with the private key used 
+for signing and the public key made available for verification. The 
+[[MULTIBASE]]/[[MULTICODEC]] representation for the public key, <code>ed25519-pub</code>,
+and the representation for the private key, <code>ed25519-priv</code>, are shown below.
+        </p>
+        <pre class="example" title="Private and Public keys for Signature">
 {
-  "seed_0": "9b937b81322d816cfab9d5a3baacc9b2a5febe4b149f126b3630f93a29527017"
+    publicKeyMultibase: "z6MkrJVnaZkeFzdQyMZu1cgjg7k1pZZ6pvBQ7XJPt4swbTQ2", 
+    privateKeyMultibase: "z3u2en7t5LR2WtQH5PfFqMqwVHBeXouLzo6haApm8XHqvjxq" 
 }
-      </pre>
-      <pre class="example">
+        </pre>
+
+        <p>
+Signing begins with a credential without an attached proof, which is converted 
+to canonical form, and then hashed, as shown in the following three examples.
+        </p>
+
+        <pre class="example" title="Credential without Proof">
 {
-  "keypair_0": {
-    "id": "https://example.com/issuer/123#key-0",
-    "type": "Ed25519VerificationKey2018",
-    "controller": "https://example.com/issuer/123",
-    "publicKeyBase58": "dbDmZLTWuEYYZNHFLKLoRkEX4sZykkSLNQLXvMUyMB1",
-    "privateKeyBase58": "47QbyJEDqmHTzsdg8xzqXD8gqKuLufYRrKWTmB7eAaWHG2EAsQ2GUyqRqWWYT15dGuag52Sf3j4hs2mu7w52mgps"
-  },
-  "keypair_1": {
-    "id": "https://example.com/issuer/123#key-0",
-    "type": "Ed25519KeyPair2020",
-    "controller": "https://example.com/issuer/123",
-    "publicKeyMultibase": "z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP",
-    "privateKeyMultibase": "zrv3kJcnBP1RpYmvNZ9jcYpKBZg41iSobWxSg3ix2U7Cp59kjwQFCT4SZTgLSL3HP8iGMdJs3nedjqYgNn6ZJmsmjRm"
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1",
+    "https://w3id.org/security/suites/ed25519-2020/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": [
+    "VerifiableCredential",
+    "UniversityDegreeCredential"
+  ],
+  "issuer": "https://example.edu/issuers/565049",
+  "issuanceDate": "2010-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
   }
 }
+        </pre>
+      
+        <pre class="example" title="Canonical Credential without Proof">
+          &lt;did:example:ebfeb1f712ebc6f1c276e12ec21&gt; &lt;https://example.org/examples#degree&gt; _:c14n0 .
+          &lt;http://example.edu/credentials/3732&gt; &lt;http://www.w3.org/1999/02/22-rdf-syntax-ns#type&gt; &lt;https://example.org/examples#UniversityDegreeCredential&gt; .
```

Comment:
this escaping makes my eyes bleed, prefer a `data-include`

Reply:
Thanks! I'm new to ReSpec. `data-include` should make bringing in the test vectors much easier.

Reply:
I think `data-include` breaks PR-preview, unless they recently fixed that... not saying we shouldn't use it, but something to consider.